N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
N3 is a bit of a misnomer since it does not parse N3: http://www.w3.org/2000/10/swap/Primer.html
Btw. as it can also parse quads, we need a quad type in RDF.